tools/admin/requests/doctor-information.ts

```diff
@@ -11,7 +11,7 @@ import {
 export type DoctorFormData = {
   firstName: string;
   lastName: string;
-  mspNumber: number | null;
+  mspNumber: string | null;
```

Comment:
I think it's no longer necessary for this to be possibly null so you can change the type of `mspNumber` to just `string`. If you could also search through the codebase and find anywhere that it was being initialized to `null`, and instead initialize it to `''` like the other fields.